pkg/domain/infra/abi/images.go

```diff
@@ -85,6 +84,110 @@ func (ir *ImageEngine) History(ctx context.Context, nameOrID string, opts entiti
 	return &history, nil
 }
 
+func (ir *ImageEngine) Mount(ctx context.Context, nameOrIDs []string, opts entities.ImageMountOptions) ([]*entities.ImageMountReport, error) {
```

Comment:
Not sure why we return a slice of pointers? Usually, we used a pointer for a single object so we can conveniently `return nil, err`. In case of a slice we can `return nil, err` as well.

No blocker, just a comment that it's rather uncommon.

Reply:
Just blindly took the code from containers mount.